CHANGELOG.md

```diff
@@ -33,6 +33,7 @@
 
 ### Features
 
+- [\#396](https://github.com/arkworks-rs/algebra/pull/396) (`ark-ec`) Add a default `mul` function to `TE/SWModelParameter` trait definition.
```

Comment:
Could you move this to the end of the section? (We have reverse chronological order)